static/js/controllers/guided-setup-modal.js

```diff
@@ -0,0 +1,151 @@
+var libphonenumber = require('libphonenumber/utils'),
+    countries = require('../modules/countries');
+
+// TODO convert this controller to use angular more and jquery less
```

Comment:
Wait, if it's not used and we won't fix it then it shouldn't be in the code. Just remove the modal completely. 

It works just fine but it's technical debt... So I still think of it as an issue. (low-priority, possibly)